core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregations.java

```diff
@@ -142,41 +80,33 @@ public static InternalAggregations reduce(List<InternalAggregations> aggregation
         return new InternalAggregations(reducedAggregations);
     }
 
-    /** The fields required to write this addAggregation to xcontent */
-    static class Fields {
-        public static final String AGGREGATIONS = "aggregations";
-    }
-
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
Specifically, moving the ToXContent part requires to adjust the Aggregation (singular) part too. The short solution is making Aggregation extend ToXContent which is not the cleanest, yet I am not sure any of the other solutions are feasible as they involve generics and complicate things quite a bit.